File: Malikah.Api/Data/Entities/Itemitemtypequantity.cs
using System;
using System.Collections.Generic;

namespace Malikah.Api.Data.Entities
{
    public partial class Itemitemtypequantity
    {
        public int Id { get; set; }
        public int? ItemId { get; set; }
        public int? ItemTypeId { get; set; }
        public int? Quantity { get; set; }
        public int? AvailableQuantity { get; set; }
    }
}

File: Malikah.Store/app/models/ItemItemTypeQuantity.php
<?php

class ItemItemTypeQuantity extends Model
{
  function __construct($id='')
  {
    parent::__construct('ID','ItemItemTypeQuantity'); //primary key = uid; tablename = users
    $this->rs['ID'] = '';
    $this->rs['ItemID'] = '';
    $this->rs['ItemTypeID'] = '';
    $this->rs['Quantity'] = '';
    $this->rs['AvailableQuantity'] = '';
    if ($id)
    {
      $this->retrieve($id);
    }
  }
}

 ?>
